jenkins-integration/jenkins-jobs/scenarios/short-test/Jenkinsfile

```diff
@@ -0,0 +1,22 @@
+node {
+    checkout scm
+
+    stage '001-create-files'
+    sh "echo 'hello!' > hello.txt"
+    sh "echo 'hello again!' > hello_again.txt"
+    sh "mkdir greetings; echo 'hi' > greetings/hi.txt"
```

Comment:
I'm a little interested in a more deeply nested test. Like 'greetings/foo/bar/baz.txt' being driven by a 'greetings/**' glob?

Reply:
Sounds good, will add.